aws/src/main/java/org/apache/iceberg/aws/glue/GlueCatalog.java

```diff
@@ -524,7 +529,11 @@ public Map<String, String> loadNamespaceMetadata(Namespace namespace)
 
   @Override
   public boolean dropNamespace(Namespace namespace) throws NamespaceNotEmptyException {
-    namespaceExists(namespace);
+    // check if namespace exists
+    if (!namespaceExists(namespace)) {
```

Comment:
As per the javadoc in the interface, if the namespace doesn't exist, `false` is returned.  I think we should return `false` instead of throwing exception to be consistent with other catalog impl

Reply:
@munendrasn Thanks for pointing that out. Made the change to return false in the case of dropNamespace.

Reply:
Yeah I think this is a reasonable behavior change according to how the interface describes the expectation:

```
   * Drop a namespace. If the namespace exists and was dropped, this will return true.
```

@jackye1995 let us know if you see any issues with this